template/src/lib/ui/stories/common/Palette.stories.js

```diff
@@ -140,6 +158,44 @@ const BackgroundTemplate = (args) => {
   );
 };
 
+const OtherTemplate = (args) => {
+  const theme = useTheme();
+  const colorDef = theme.palette.other;
+  const textColor = theme.palette.common.white;
+  return (
+    <Grid container>
+      <Box style={{ backgroundColor: colorDef.tooltip, ...boxStyle }} {...args}>
+        <Typography variant='caption' style={{ color: textColor }}>
+          Tooltip
+          <br />
+          {colorDef.tooltip}
+        </Typography>
+      </Box>
+      <Box style={{ backgroundColor: colorDef.snackbar, ...boxStyle }} {...args}>
```

Comment:
I see lots of custom colors, do we really need them?

Reply:
see https://github.com/CartoDB/cra-template-carto/pull/86#discussion_r526024246